.rubocop.yml

```diff
@@ -430,6 +430,8 @@ Style/PerlBackrefs:
 
 Style/PredicateName:
   Enabled: True
+  Exclude:
```

Comment:
We should not exclude the whole file, but you can disable a specific cop. Just take a look at the end of this documentation.
https://github.com/bbatsov/rubocop/blob/master/manual/configuration.md